src/index.html

```diff
@@ -9,6 +9,8 @@
     <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Colored blocks</h1>
+    <div class="block1"></div>
+    <div class="block2"></div>
+    <div class="block3"></div>
```

Comment:
names are not descriptive
https://github.com/mate-academy/layout_colored-blocks/blob/master/checklist.md

Reply:
`block-red`, `block-green` are better